AcademyResidentInformationApi.Tests/V1/Helper/TestHelper.cs

```diff
@@ -86,14 +88,16 @@ public static Occupation CreateDatabaseOccupationEntityForCouncilProperty(int ac
             return cto;
         }
 
-        public static Email CreateDatabaseEmailAddressForTaxPayer(int accountRef, string email = null)
+        public static Email CreateDatabaseEmailAddressForTaxPayer(int accountRef, List<string> email = null)
```

Comment:
I wonder if there's a way to create multiple email database records so that each record has a separate email address from the email list we provide, rather than just saving the first email from the list to a single record?

Reply:
I'm looking at the infrastructure for email, and I'm only seeing a single email column in there, compared with PhoneNumber where there are columns for phone number 1 - 4, which is why I only had the single email present. 

Adapting this code to look a bit more like the PhoneNumber list is very possible, but I'm not sure if that is reflective of what we get back from the database.